src/frontends/sql/decode.cc

```diff
@@ -123,4 +124,10 @@ Value DecodeExpressionArena(const ExpressionArena &expr_arena,
   return *top_level_value;
 }
 
+Value DecodeExpressionArena(const ExpressionArena &expr_arena) {
+  ir::IRContext ir_context;
+  DecoderContext decoder_context(ir_context);
+  return DecodeExpressionArena(expr_arena, decoder_context);
```

Comment:
Won't all the operators in `Value` have dangling references when DecoderContext is destroyed?